File: Tareas/Unidad2/a2253332351_practica04/33_AreaTrapecio.cpp
#include <iostream>
using namespace std;

int main() {
    double baseMayor;
    double baseMenor;
    double altura;
    double area;

    cout << "Ingrese la base mayor del trapecio: ";
    cin >> baseMayor;
    cout << "Ingrese la base menor del trapecio: ";
    cin >> baseMenor;
    cout << "Ingrese la altura del trapecio: ";
    cin >> altura;

    area = ((baseMayor + baseMenor) * altura) / 2;

    cout << "El área del trapecio es: " << area << endl;

    return 0;
}

File: Tareas/Unidad2/a2253332351_practica04/33_AreaTrapecio.java
import java.util.Scanner;

public class AreaTrapecio {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);
        double baseMayor;
        double baseMenor;
        double altura;
        double area;

        System.out.print("Ingrese la base mayor del trapecio: ");
        baseMayor = entrada.nextDouble();
        System.out.print("Ingrese la base menor del trapecio: ");
        baseMenor = entrada.nextDouble();
        System.out.print("Ingrese la altura del trapecio: ");
        altura = entrada.nextDouble();

        area = ((baseMayor + baseMenor) * altura) / 2;

        System.out.println("El área del trapecio es: " + area);
    }
}
